docs/quickstarts/nextjs.mdx

```diff
@@ -277,83 +268,116 @@ export default function Example() {
 
 ### Server side
 
-#### API Routes
+#### App Router
 
-You can protect your API routes by using the [`getAuth`](/docs/references/nextjs/get-auth) helper and retrieve data from your own systems.
+[`auth()`][auth-ref] and [`currentUser()`][currentuser-ref] are App Router-specific helpers that you can use inside of your Route Handlers, middleware, Server Components, and Server Actions.

```

Comment:
nit: Middleware is capitalized in the Next.js docs 👀 

```suggestion
[`auth()`][auth-ref] and [`currentUser()`][currentuser-ref] are App Router-specific helpers that you can use inside of your Route Handlers, Middleware, Server Components, and Server Actions.

```

Reply:
I don't mind this change, but we use 'middleware' everywhere in the docs so we should probably do a docs wide pass.